Lib/test/test_csv.py

```diff
@@ -1423,6 +1448,11 @@ def test_delimiters(self):
         self.assertEqual(dialect.quotechar, "'")
         dialect = sniffer.sniff(self.sample14)
         self.assertEqual(dialect.delimiter, '\0')
+        self.assertRaisesRegex(csv.Error, "Could not determine delimiter",
+                               sniffer.sniff, self.sample15)
+        self.assertRaisesRegex(csv.Error, "Could not determine delimiter",
+                               sniffer.sniff, self.sample16)
+
```

Comment:
```suggestion
                               sniffer.sniff, self.sample16)
```

Double blank lines are to separate classes, not methods (see PEP-8).